test/specs/app.spec.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
```

Comment:
I like the idea... but I also like having them in the spec file so that you can see what is available per spec. We may not need to expose each for every spec.